.github/scripts/build_a12.sh

```diff
@@ -0,0 +1,51 @@
+build_from_image(){
+	export TITLE=kernel-aarch64-$(echo $1 | sed 's/Image-//g')
+	echo "[+] title: $TITLE"
+
+	export PATCH_LEVEL=$(echo $1 | awk -F_ '{ print $2}')
+	echo "[+] patch level: $PATCH_LEVEL"
+
+	echo "[+] Download prebuilt ramdisk"
+	curl -Lo gki-kernel.zip https://dl.google.com/android/gki/gki-certified-boot-android12-5.10-$(PATCH_LEVEL)_r1.zip
```

Comment:
should be `$PATCH_LEVEL

Reply:
$PATCH_LEVEL will add _r1 to it

Reply:
`${PATCH_LEVEL}`? `$(PATCH_LEVEL) is a subprocess call